### PR TITLE
dockertools/nsenterexec: fix err shadow

### PR DESCRIPTION
**Pull request by euank**
The shadow of err meant the combination of `exec-handler=nsenter` +
`tty` + a non-zero exit code meant that the exit code would be LOST
FOREVER :ghost:

This isn't all that important since no one really used the nsenter exec
handler as I understand it

```release-note
NONE
```

### CODE REVIEW

**Comment by euank** [comment]
Shoutout to @sjpotter for noticing this a while back!

File: cluster/gce/gci/configure.sh [hunk 1]
```diff
@@ -115,7 +115,7 @@ function split-commas {
 }
 
 function install-gci-mounter-tools {
-    local -r rkt_version="v1.18.0"
+    local -r rkt_version="v1.22.0"
```
Comment (yujuhong) [inline cluster/gce/gci/configure.sh]: The change seems unrelated.

File: pkg/kubelet/dockertools/exec.go [hunk 2]
```diff
@@ -54,7 +54,8 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 	args = append(args, cmd...)
 	command := exec.Command(nsenter, args...)
 	if tty {
-		p, err := kubecontainer.StartPty(command)
+		var p *os.File
+		p, err = kubecontainer.StartPty(command)
```
Comment (yujuhong) [inline pkg/kubelet/dockertools/exec.go]: How about declaring a new variable (e.g., `var commandErr error`) before line 56 to reduce the chance of regression again?

File: cluster/gce/gci/configure.sh (hunk 1 above)
Comment (euank) [inline cluster/gce/gci/configure.sh]: Oops! `git commit -a` strikes! I'll fix that

File: pkg/kubelet/dockertools/exec.go (hunk 2 above)
Comment (euank) [inline pkg/kubelet/dockertools/exec.go]: Sure, though I figured the next time this code was touched would be to delete it anyways 😸

Comment (yujuhong) [inline pkg/kubelet/dockertools/exec.go]: That's my hope, but who knows...

**Comment by yujuhong** [comment]
@k8s-bot cvm gke e2e test this.
@k8s-bot gci gke e2e test this
